src/inet/TCPEndPoint.cpp

```diff
@@ -2368,15 +2372,15 @@ void TCPEndPoint::HandlePendingIO()
     // ready to be received on the socket, process the incoming connection.
     if (State == kState_Listening)
     {
-        if (OnConnectionReceived != nullptr && mPendingIO.IsReadable())
+        if (OnConnectionReceived != nullptr && mPendingIO.Has(System::SocketEventFlag::kRead))
             HandleIncomingConnection();
     }
 
     // If in the processes of initiating a connection...
     else if (State == kState_Connecting)
     {
         // The socket being writable indicates the connection has completed (successfully or otherwise).
-        if (mPendingIO.IsWriteable())
+        if (mPendingIO.Has(System::SocketEventFlag::kWrite))
```

Comment:
Any reason to not make `IsWriteable()` as a convenience inline for `Has(System::SocketEventFlag::kWrite)`?